### PR TITLE
deprecate three admission controller

### PR DESCRIPTION
**Pull request by CaoShuFeng**
The related code:
https://github.com/kubernetes/kubernetes/blob/317853c90c674920bfbbdac54fe66092ddc9f15f/pkg/kubeapiserver/options/plugins.go#L102

/cc @hzxuzhonghu  for tech review.
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7363)

### CODE REVIEW

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -418,15 +418,15 @@ subresource of the referenced *owner* can change it.
...
+### PersistentVolumeLabel (DEPRECATED)
 
 This admission controller automatically attaches region or zone labels to PersistentVolumes
 as defined by the cloud provider (for example, GCE or AWS).
 It helps ensure the Pods and the PersistentVolumes mounted are in the same
 region and/or zone.
 If the admission controller doesn't support automatic labelling your PersistentVolumes, you
 may need to add the labels manually to prevent pods from mounting volumes from
-a different zone.
+a different zone. PersistentVolumeLabel is DEPRECATED in favor of NewPersistentVolumeLabelController in CCM.
```
Comment (tengqm) [inline docs/admin/admission-controllers.md]: What is `NewPersistentVolumeLabelController` ?

Comment (hzxuzhonghu) [reply docs/admin/admission-controllers.md]: pkg/controller/cloud/pvlcontroller.go

**Comment by hzxuzhonghu** [comment]
LGTM

File: docs/admin/admission-controllers.md [hunk 2]
```diff
@@ -418,15 +418,15 @@ subresource of the referenced *owner* can change it.
...
+### PersistentVolumeLabel (DEPRECATED)
 
 This admission controller automatically attaches region or zone labels to PersistentVolumes
 as defined by the cloud provider (for example, GCE or AWS).
 It helps ensure the Pods and the PersistentVolumes mounted are in the same
 region and/or zone.
 If the admission controller doesn't support automatic labelling your PersistentVolumes, you
 may need to add the labels manually to prevent pods from mounting volumes from
-a different zone.
+a different zone. PersistentVolumeLabel is DEPRECATED in favor of PersistentVolumeLabelController in [cloud controller manager](/docs/tasks/administer-cluster/running-cloud-controller/).
```
Comment (tengqm) [inline docs/admin/admission-controllers.md]: Thanks for the revision. In the revised version, `PersistentVolumeLableController` is still confusing to users (readers). The page pointed did mentioned new ways of labelling persistent volumes, however, it is about creating and using Initializers. Could you please revise this again to make sure users (not necessarily Go developers) to understand what needs to be done?

Comment (CaoShuFeng) [reply docs/admin/admission-controllers.md]: What about current version?

Description about PersistentVolumeLableController is removed. But `cloud conroller manager` is keeped.

**Pull request merged**